DragonKitchen/DragonKitchen/Util/ViewAssets/NavigationBar.swift

```diff
@@ -12,14 +12,22 @@ struct NavigationBar: View {
     @State var pageText: Int = 1
     @State var isOn = [true, false, false, false, false]
     @Binding var isCleared: Bool
+    @EnvironmentObject var CurrentPage: CurrentPageModel
```

Comment:
변수이름도 lowerCamelCase가 일반적인 것 같아요 !

```suggestion
    @EnvironmentObject var currentPage: CurrentPageModel
```

Reply:
감사합니닷!!!